extensions/data-transfer/portability-data-transfer-flickr/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     compile project(':portability-spi-transfer')
 
     compile("com.flickr4java:flickr4java:${flickrVersion}")
+    compile("org.scribe:scribe:1.3.7")
```

Comment:
The new version of flickr4java requires Scribe 6.9.0. Why is this dependency being added here?